src/app/tests/suites/certification/Test_TC_PRS_2_1.yaml

```diff
@@ -34,199 +34,84 @@ tests:
-      attribute: "MeasuredValue"
-      response:
-          value: 0
+              minValue: -32768
+              maxValue: 32767
 
     - label: "Read the mandatory attribute constraints: MinMeasuredValue"
       command: "readAttribute"
       attribute: "MinMeasuredValue"
       response:
           constraints:
               type: int16
-
-    - label: "Write the default values to mandatory attribute: MinMeasuredValue"
-      command: "writeAttribute"
-      attribute: "MinMeasuredValue"
-      arguments:
-          value: 0
-      response:
-          error: UNSUPPORTED_WRITE
-
-    - label: "Reads back mandatory attribute: MinMeasuredValue"
-      command: "readAttribute"
-      attribute: "MinMeasuredValue"
-      response:
-          value: 0
+              minValue: -32768
+              maxValue: 32767
 
     - label: "Read the mandatory attribute constraints: MaxMeasuredValue"
       command: "readAttribute"
       attribute: "MaxMeasuredValue"
       response:
           constraints:
               type: int16
+              minValue: -32768
+              maxValue: 32767
 
-    - label: "Write the default values to mandatory attribute: MaxMeasuredValue"
-      command: "writeAttribute"
-      attribute: "MaxMeasuredValue"
-      arguments:
-          value: 0
-      response:
-          error: UNSUPPORTED_WRITE
-
-    - label: "Reads back mandatory attribute: MaxMeasuredValue"
-      command: "readAttribute"
-      attribute: "MaxMeasuredValue"
-      response:
-          value: 0
-
-    #disabled due to issue #13442
+    #Issue 18243
     - label: "Read the optional attribute: Tolerance"
-      disabled: true
-      command: "readAttribute"
-      attribute: "Tolerance"
-      response:
-          value: 0
-
-    - label: "Write the default values to optional attribute: Tolerance"
-      disabled: true
-      command: "writeAttribute"
-      attribute: "Tolerance"
-      arguments:
-          value: 0
-      response:
-          error: UNSUPPORTED_WRITE
-
-    - label: "Reads back optional attribute: Tolerance"
-      disabled: true
+      PICS: PICS_SKIP_SAMPLE_APP
       command: "readAttribute"
       attribute: "Tolerance"
       response:
-          value: 0
+          constraints:
+              type: int16
+              minValue: 0
+              maxValue: 2048
 
-    #Following attributes are based on the value of the extended feature in the cluster feature map
     - label: "Read the optional attribute: ScaledValue"
-      disabled: true
-      command: "readAttribute"
-      attribute: "ScaledValue"
-      response:
-          value: 0
-
-    - label: "Write the default values to optional attribute: ScaledValue"
-      disabled: true
-      command: "writeAttribute"
-      attribute: "ScaledValue"
-      arguments:
-          value: 0
-      response:
-          error: UNSUPPORTED_WRITE
-
-    - label: "Reads back optional attribute: ScaledValue"
-      disabled: true
+      PICS: PICS_SKIP_SAMPLE_APP
```

Comment:
Why is this being skipped?

Reply:
optional attribute showing status = 0x86 (UNSUPPORTED_ATTRIBUTE) when executed
Issue: https://github.com/project-chip/connectedhomeip/issues/18243

Reply:
But this steps are works fine with optional: true tag i will update these steps